kafka/producer_test.go

```diff
@@ -334,7 +334,7 @@ func TestProducerConcurrentClose(t *testing.T) {
 	wg.Wait()
 }
 
-func newClusterWithTopics(t testing.TB, topics ...apmqueue.Topic) (*kgo.Client, []string) {
+func newClusterWithTopics(t testing.TB, partitions int32, topics ...apmqueue.Topic) (*kgo.Client, []string) {
```

Comment:
Where are we using the `partitions`?

Reply:
shame on me 🤦🏼, I pushed a fix.

Reply:
![shame](https://media.tenor.com/35NdUtUyqdQAAAAM/shame-no-sense.gif)